src/nupic/research/fast_temporal_memory.py

```diff
@@ -36,8 +36,14 @@ class FastTemporalMemory(TemporalMemory):
   """
 
   def __init__(self, *args, **kwargs):
+    maxSegmentsPerCell = kwargs.get("maxSegmentsPerCell") or 255
```

Comment:
`get` has an optional second parameter that is the default if the key isn't found.